public/components/visualize/components/sample-data.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Wazuh app - React component for Visualize.
+ * Copyright (C) 2015-2021 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+import { EuiCallOut, EuiLink } from '@elastic/eui';
+import { UI_ERROR_SEVERITIES } from '../../../react-services/error-orchestrator/types';
+import { UI_LOGGER_LEVELS } from '../../../../common/constants';
+import React, { useState, useEffect } from 'react';
+import { WzRequest } from '../../../react-services';
+import { getErrorOrchestrator } from '../../../react-services/common-services';
+
+export const SampleData = ({ context = 'sample-data', ...props }) => {
+  const [isSampleData, setIsSampleData] = useState(false);
+
+  const usesSampleData = async () => {
+    try {
+      const result = (await WzRequest.genericReq('GET', '/elastic/samplealerts', {})).data
+        .sampleAlertsInstalled;
+      setIsSampleData(result);
+    } catch (error) {
+      const options = {
+        context,
+        level: UI_LOGGER_LEVELS.ERROR,
+        severity: UI_ERROR_SEVERITIES.UI,
+        error: {
+          error: error,
+          message: error.message || error,
+          title: error.name || error,
+        },
+      };
+      getErrorOrchestrator().handleError(options);
+    }
+  };
+  useEffect(() => {
+    usesSampleData();
+  }, [
+    setIsSampleData,
+    context,
+    UI_ERROR_SEVERITIES,
+    UI_LOGGER_LEVELS,
+    getErrorOrchestrator,
+    WzRequest,
```

Comment:
What is the purpose of having this here?

Reply:
In order to be thorough, none of these dependencies should change in theory, but since they are elements from outside the `useEffect` call I added them as dependencies.